SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/UserSpanId.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "UserSpanId.generated.h"
+
+USTRUCT(Blueprintable)
+struct FUserSpanId
+{
+	GENERATED_BODY()
+
+	FUserSpanId() { Data.Reserve(16); }
```

Comment:
Sometimes these are not actually used so think this might be counter-intuitive to allocate memory for an empty span

Reply:
Perhaps. My use case for this is the SpanIdToUserSpanId. I create a default FUserSpanId then populate it's data in a loop. Ideally I would reserve the "Data". I could do that in the conversion function itself instead. Don't really mind.

Reply:
Yeah I think that makes sense, there are places where the reservation isn't needed!